### PR TITLE
VAGOV-2802: Open location menu when on local facility

### PR DESCRIPTION
**Pull request by acabouet**
## Description
Fixes behavior of facilities side nav. 

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by ethanteague — APPROVED** [review]
LGTM :+1:

File: src/site/filters/liquid.js [hunk 1]
```diff
@@ -208,8 +212,16 @@ module.exports = function registerFilters() {
 
   // check is this is a root level page
   liquid.filters.isRootPage = path => {
+    const isFacilityRoot = /^(?:\/pittsburgh-health-care)+$|^(?:\/pittsburgh-health-care)\/((?!stories|events|locations|press-releases|health-services|jobs-careers).)*$/;
```
Comment (jbalboni) [inline src/site/filters/liquid.js]: This doesn't seem like a sustainable approach as we add more regions.

**Review by jbalboni — APPROVED** [review]
I'm a little concerned about the amount of special case logic we're putting in for the Pittsburgh region, but looks good otherwise.

File: src/site/filters/liquid.js (hunk 1 above)
Comment (acabouet) [reply src/site/filters/liquid.js]: yeah I agree. I want to check in with the content/migration team and see if there's a nailed down pattern for facility region names. If there is, this regex can be updated to be less hardcoded. I also want to try out importing paths that should be treated as root from a separate helper file and just looping through them to build this regex in a future sprint.

**Pull request merged**